app/controllers/api/pages_controller.rb

```diff
@@ -58,6 +58,16 @@ def similar
     render :index, format: :json
   end
 
+  def total_donations
+    @page = Page.find(params[:page_id])
+    if @page.campaign.blank?
+      total_donations = FundingCounter.convert(page: @page, currency: params[:currency])
+    else
+      total_donations = FundingCounter.convert_multiple(pages: @page.campaign.pages, currency: params[:currency])
```

Comment:
Instead of having the method on the Campaign model as we do for the actions counter, I decided to add a method to `FundingCounter` that takes an iterable collection of pages because I remembered the use case where we might want to be able to pass multiple page IDs, not necessarily just the campaign - the endpoint won't support that out of the box but the logic is there if we want to switch to it.